Skip those tests which require instaling the HuggingFace transformers pip package and likely takes some time.

Author: Alexander Grund (TU Dresden)

diff --git a/test/inductor/test_deterministic.py b/test/inductor/test_deterministic.py
--- a/test/inductor/test_deterministic.py
+++ b/test/inductor/test_deterministic.py
@@ -167,7 +167,7 @@ class DeterministicTest(TestCase):
             torch.testing.assert_close(eager, compiled_out)
 
     @skipIfRocm(msg="https://github.com/pytorch/pytorch/issues/180681")
-    @unittest.skipIf(IS_FBCODE, "Skipping run2run determinism test in fbcode")
+    @unittest.skip("Requires transformers pip package")
     @parametrize("model_name", ["GoogleFnet", "BertForMaskedLM", "DistillGPT2"])
     @parametrize("training_or_inference", ["training", "inference"])
     @parametrize("precision", ["float32", "bfloat16", "float16", "amp"])
